piezo_web_app/PiezoWebApp/tests/integration_tests/submit_job_test.py

```diff
@@ -71,7 +71,13 @@ def test_correct_python_job_is_submitted_correctly(self):
                 'imagePullPolicy': 'Always',
                 'mainApplicationFile': '/path_to/file',
                 'sparkVersion': '2.4.0',
-                'restartPolicy': {'type': 'Never'},
+                'restartPolicy': {
+                    'type': 'Never',
```

Comment:
Just for clarity `Never` should probably be changed to `onFailure`

Same for following two

Reply:
11aadbd